ci/docker/openstack-cpi-release/Dockerfile

```
@@ -1,56 +1,29 @@
-    cd /tmp/ruby-install && \
-    curl https://codeload.github.com/postmodern/ruby-install/tar.gz/v0.8.3 | tar -xz && \
-    cd /tmp/ruby-install/ruby-install-0.8.3 && \
-    make install && \
-    rm -rf /tmp/ruby-install
-
-#Ruby
-RUN ruby-install --system ruby 3.1.0
-
-#BOSH CLI
-RUN curl -sL https://github.com/cloudfoundry/bosh-cli/releases/download/v${bosh_cli_version}/bosh-cli-${bosh_cli_version}-linux-amd64 \
+FROM ubuntu
```

Hadolint (2.15.1)

[warning] Always tag the version of an image explicitly

(DL3006)